src/models/converters/data_decoded.rs

```diff
@@ -0,0 +1,54 @@
+use crate::models::commons::DataDecoded;
+use crate::models::service::transactions::SettingsInfo;
+use crate::utils::SETTINGS_CHANGE_METHODS;
+
+impl DataDecoded {
+    pub(super) fn to_settings_info(&self) -> Option<SettingsInfo> {
+        match self.method.as_str() {
+            method_name if method_name == SETTINGS_CHANGE_METHODS[0] => {
```

Comment:
this works for now, but I would have extracted the string inside the `SETTINGS_CHANGE_METHODS` array into separate constants and use them here. Now if we change the order in the `SETTINGS_CHANGE_METHODS` array it will break

Reply:
I will address it